app/utils/hentPath.ts

```diff
@@ -3,13 +3,16 @@ import { EYtelse } from '~/typer/ytelse';
 
 const URL_BARNETRYGD = '/ba';
 const URL_KONTANTSTØTTE = '/ks';
+const URL_DOKUMENTASJON = '/doc';
```

Comment:
Hva er det denne url-en er for? Denne innebærer ble t vi har en egen /doc utenfor ks og ba - stemmer det? 🕵️

Reply:
Denne bør kunne slettes